sormas-backend/utils/importUsersExample.sql

```diff
@@ -0,0 +1,81 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2020 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+  The following script provides an example of how to import a list of users into the database directly with a valid password, role and region.
+
+  In this script, users are imported with the following information:
+  - username
+  - password
+  - firstname
+  - lastname
+  - region
+  - roles: CASE_SUPERVISOR, CONTACT_SUPERVISOR, SURVEILLANCE_SUPERVISOR
+
+  Certain roles require multiple other fields in order to work properly (e.g. district or facility).
+
+
+  In order to run the script the following extensions have to be enabled by the root user:
+  CREATE EXTENSION pgcrypto;
+  CREATE EXTENSION "uuid-ossp";
+
+  Also make sure to replace the ROLLBACK statement at the end after testing that your script is correct and you are ready to run it.
+*/
+
+BEGIN;
+
+CREATE OR REPLACE FUNCTION pg_temp.insert_user(username varchar, pass varchar, firstname varchar, lastname varchar,
+                                               region_epid_code varchar)
+    RETURNS integer
+    LANGUAGE plpgsql AS
+$$
+DECLARE
+    inserted_id integer;
+    salt        varchar;
+    region_id   bigint;
+BEGIN
+
+    inserted_id := (SELECT max(id) FROM users) + 1;
+    salt := gen_salt('md5');
```

Comment:
SORMAS uses here also 'sha256', why using 'md5' here?

Reply:
https://www.postgresql.org/docs/9.6/pgcrypto.html `gen_salt` method doesn't support the sha256 algorithm for generating the salt. The only supported ones are `bf`, `md5`, `xdes` and `des`.
`bf` would be the best option, however the string it generates exceeds the supported column length.
Since this refers only to the salt, I think the `md5` option is still a pretty good one. 

This is an example and can be tweaked by anyone who intends to use it.

Reply:
Okay, understood